weave/tests/trace/test_client_trace.py

```diff
@@ -2664,6 +2664,42 @@ def return_nested_object(nested_obj: NestedObject):
     assert call_result.output == nested_ref.uri()
 
 
+@pytest.mark.parametrize("batch_size", [1, 11, 101, 120])
+def test_calls_stream_column_expansion_dynamic_batch_size(client, batch_size):
+    @weave.op
+    def test_op(x):
+        return x
+
+    for i in range(batch_size):
+        test_op(i)
+
+    res = client.server.calls_query_stream(
+        tsi.CallsQueryReq(
+            project_id=client._project_id(),
+            columns=["output"],
+            expand_columns=["output"],
+        )
+    )
+    calls = list(res)
+    assert len(calls) == batch_size
+    for i in range(batch_size):
+        assert calls[i].output == i
+
+    res = client.server.calls_query_stream(
+        tsi.CallsQueryReq(
+            project_id=client._project_id(),
+            columns=["output"],
+            expand_columns=[
+                "output.a.b.c.d.e.f.g.h.i.j.k.l.m.n.o.p.q.r.s.t.u.v.w.x.y.z"
```

Comment:
I don't understand what this is for.  How does it relate to this test?

Reply:
Hmm, probably not necessary now, the previous calculation accounted for the depth of the nested ref, so in this case it would be depth 27, which would have affected the batching. now it doesn't, but if this test were written in the first place it would have triggered the bug. Ill delete it.